### PR TITLE
Updating/Fixing grammatical errors

### PR DESCRIPTION
**Pull request by roadev**
Updating/fixing grammatical errors at spanish translations (for basic guides).

### CODE REVIEW

File: es/lessons/basics/control-structures.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ iex> cond do
 "But this will"
 ```
 
-Como `case`, `cond` alzará un error si no hay una coincidencia. Para manejar esto, podemos definir una condición cuyo valor es `true`:
+Como `case`, `cond` alzará un error si no hay una coincidencia, para manejar esto, podemos definir una condición cuyo valor es `true`:
```
Comment (setzer777) [inline es/lessons/basics/control-structures.md]: I don't like this improvement, it sounds super awkward.
How about this one: 
```
`case` y `cond` alzarán un error si no hay una coincidencia. Para manejarlos, podemos definir una condición cuyo valor es `true`:
```

Just saying, let's debate it a little more.

**Comment by setzer777** [comment]
I really loved the improvements. I'm a super gramatical nazi, and I think this will really help to the content reputation.
But Jesus, there were a lot of corrections! 
@doomspork @pjhampton can you inform the other spanish translators to give it a quick review? c:

You have my like @roadev :+1:

File: es/lessons/basics/control-structures.md (hunk 1 above)
Comment (roadev) [inline es/lessons/basics/control-structures.md]: You're right. It sounds better with a dot instead of a comma.

**Comment by roadev** [comment]
Thanks :)
Some friends call me also a grammatical nazi... haha
The grammar is extremely important :D !
@jesusangelm <https://github.com/jesusangelm>, @erickgnavar
<https://github.com/erickgnavar>, @milmazz <https://github.com/milmazz>

On 6 April 2016 at 00:13, Alberto Romero <notifications@github.com> wrote:

> I really loved the improvements. I'm a super gramatical nazi, and I think
> this will really help to the content reputation.
> But Jesus, there were a lot of corrections!
> @doomspork <https://github.com/doomspork> @pjhampton
> <https://github.com/pjhampton> can you inform the other spanish
> translators to give it a quick review? c:
>
> You have my like @roadev <https://github.com/roadev> [image: :+1:]
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/doomspork/elixir-school/pull/261#issuecomment-206121186>
>



-- 
*Juan David Roa*
CTO at CreaDev <http://creadev.co>
Personal Blog
JuanRoa.me
Ingeniería de Sistemas
Universidad Cooperativa de Colombia

**Comment by roadev** [comment]
Thanks :) 
Some friends call me also a grammatical nazi... haha.
The grammar is extremely important :D ! }
@jesusangelm, @erickgnavar, @milmazz

**Comment by doomspork** [comment]
Is this good to merge @jesusangelm, @erickgnavar, or @milmazz?

**Comment by jesusangelm** [comment]
Looks great for me!

**Comment by doomspork** [comment]
Thank you @jesusangelm and @roadev!

**Comment by roadev** [comment]
You're welcome guys! :D

**Pull request merged**